include/ck/wrapper/tensor.hpp

```diff
@@ -275,21 +315,30 @@ struct Tensor
         return this->operator[](make_tuple(idxs...));
     }
 
-    // Getter for the value reference
+    /**
+     * \brief Getter for the reference.
```

Comment:
```suggestion
     * \brief Getter of tensor value reference.
```
Additionally please add information on input argument restrictions.